batch/batch/worker/worker.py

```diff
@@ -1306,7 +1274,7 @@ def __init__(
 
         requester_pays_project = job_spec.get('requester_pays_project')
 
-        self.timings = Timings(lambda: False)
```

Comment:
Ah great. I was worried that `DockerJob` would depend on its `deleted` field here. The fact that it doesn't makes me more at ease.